src/instructions/create_instructions.py

```
@@ -1,54 +1,82 @@
-        - If the previous row does not advance to the next node (diff [0, 1], binary 1) this edge will be part of an
-            'inc' stitch (one old node fans into multiple new nodes)
-        - If the next row does not advance to the next node (diff [1, 0], binary 2) this edge will be part of a 'dec'
-            stitch (multiple old nodes collapse into one new node)
-        - All stitches start with an advancement of both rows to the next node (diff [1, 1], binary 3). If there are no
-            'inc' or 'dec' stitches after - this edge will represent a 'sc' stitch
+def dtw_to_stitches(dtw_path: np.ndarray, creases: Optional[np.ndarray] = None) -> list[str]:
+    """Convert a DTW alignment path between two crochet rows into a list of stitch instructions.
 
-    We will represent those edges as a string (where each diff is encoded as a digit: d_prev × 2 + d_next)
-    and split the resulting string to individual stitches.
+    Each step in the path is encoded as a digit (d_prev × 2 + d_next):
```

Ruff (0.15.20)

[warning] Docstring contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF002)